build/update_api.sh

```diff
@@ -28,8 +30,9 @@ if [[ ! -d "${DIRECTORY}" ]]; then
   unzip "${FILE}" -d "${DIRECTORY}"
 fi
 
-APIS=`cat ${DIRECTORY}/resources/app/product.json | jq -r '.extensionEnabledApiProposals'`
+APIS=$( jq -r '.extensionEnabledApiProposals' "${DIRECTORY}/resources/app/product.json" )
 
-APIS=`echo "${APIS}" | jq '. += {"jeanp413.open-remote-ssh": ["resolvers", "tunnels", "terminalDataWriteEvent", "contribRemoteHelp", "contribViewsRemote"]}'`
+APIS=$( echo "${APIS}" | jq '. += {"jeanp413.open-remote-ssh": ["resolvers", "tunnels", "terminalDataWriteEvent", "contribRemoteHelp", "contribViewsRemote"]}' )
 
-cat <<< $(jq --argjson v "${APIS}" 'setpath(["extensionEnabledApiProposals"]; $v)' product.json) > product.json
+jsonTmp=$(jq --argjson v "${APIS}" 'setpath(["extensionEnabledApiProposals"]; $v)' product.json)
```

Comment:
Ah! Missing spaces...

Reply:
Dang, I thought I caught all those. I have another question about this file. I'm not able to get `URL=$( curl -s "https://update.code.visualstudio.com/api/update/win32-x64-archive/${VSCODE_QUALITY}/VERSION" | jq -c '.url' | sed -E 's/.*"([^"]+)".*/\1/' )` to pull the URL with either `stable` or `insider`. It might be a network issue, but I'd like to replace the following `sed` commands in the assignment of `FILE` & `DIRECTORY` with bash expansions.

Reply:
What am I missing?
```bash
$ curl -s "https://update.code.visualstudio.com/api/update/win32-x64-archive/stable/VERSION"
<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<title>Error</title>
</head>
<body>
<pre>Cannot GET /api/update/win32-x64-archive/stable/VERSION</pre>
</body>
</html>

$ curl -s "https://update.code.visualstudio.com/api/update/win32-x64-archive/insider/VERSION"
<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<title>Error</title>
</head>
<body>
<pre>Cannot GET /api/update/win32-x64-archive/insider/VERSION</pre>
</body>
</html>
```

Reply:
Ya, I've fixed that error in https://github.com/VSCodium/vscodium/commit/9a210dc96425be77fe85489347a76d975c87c211 (on the insider branch)
You can just replace the `VERSION` with `0000000000000000000000000000000000000000`

Since last week, MS must require a valid UUID.

Reply:
OK, yes that works. I'll change it to:
```bash
URL=$( curl -s "https://update.code.visualstudio.com/api/update/win32-x64-archive/${VSCODE_QUALITY}/VERSION" | jq -c '.url' | sed -E 's/.*"([^"]+)".*/\1/' )
# echo "url: ${URL}"
FILE="${URL##*/}"
# echo "file: ${FILE}"
DIRECTORY="${FILE%.zip}"
# echo "directory: ${DIRECTORY}"
```
Assuming you want the debug echos left in?

Reply:
Yes, it is